service.watchedlist/resources/language/English/strings.po

```diff
@@ -0,0 +1,392 @@
+# XBMC Media Center language file
+# Addon Name: WatchedList
+# Addon id: service.watchedlist
+# Addon version: 1.0.5
+# Addon Provider: schapplm
+msgid ""
+msgstr ""
+"Project-Id-Version: XBMC-Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: 2016-03-28 16:47+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+msgctxt "Addon Summary"
+msgid "Save watched Movies and TV Episodes independent from filenames"
+msgstr ""
+
+msgctxt "Addon Description"
+msgid "\n		Export: Searches the XBMC-Database for watched files. Determine imdb-id and thetvdb-id to identify movies and TV-episodes. Then save the list to a new independent table.\n		Import: Set the watched state for each video file in xbmc.\n		Automatic background process without user interaction."
+msgstr ""
+
+msgctxt "Addon Disclaimer"
+msgid "This script does not regard the file format. A Movie watched in DVD quality will also set	the 1080p version as watched."
```

Comment:
Also here, please just add plain text. Formatting shouldnt be done by language file.